packages/lib/src/acting-io.ts

```diff
@@ -42,9 +42,14 @@ export async function selectAndActOnGeneric<ID, O, L>(command: SelectingCommand<
 		chosenId = await getIdFromUser(command, list)
 	}
 	const updatedItem = await actionFunction(chosenId)
+
+	if (successMessage) {
+		process.stdout.write(`${successMessage.replace('{{id}}', JSON.stringify(chosenId))}\n`)
+	}
+
 	return [chosenId, updatedItem]
 }
-selectActOnAndOutputGeneric.flags = outputList.flags
+selectAndActOnGeneric.flags = outputList.flags
```

Comment:
I don't think this should be here. The flags from `outputList.flags` all pertain to complex output that can be formatted as JSON or YAML which does not exist here.

Reply:
I thought this was a typo since there is two occurrences of `selectActOnAndOutputGeneric.flags = outputList.flags` in this module and the other is directly under `selectActOnAndOutputGeneric`.

I have gone through and removed setting flags for this and `selectAndActOn` and instead use `APICommand.flags` where necessary.

Reply:
Oops...yeah, that shouldn't have been there at all. It's nice that I can find my bugs in your reviews so I can blame you for them. :-)